config.json

```diff
@@ -16,6 +16,17 @@
       ]
     },
     {
+      "uuid": "b02a4214-0f1a-9480-2132-2ea3d5073dbfbec0aa4",
+      "slug": "two-fer",
+      "core": false,
+      "unlocked_by": null,
+      "difficulty": 1,
+      "topics": [
+        "Control-flow (conditionals)",
+        "Strings"
+      ]
+    },
```

Comment:
New exercises should be added at the end of the list please.
Before the deprecated entry for 'octal'